Puppet/PuppetUITests/AnalyticsUITests.swift

```diff
@@ -4,6 +4,10 @@ class AnalyticsUITests: XCTestCase {
   private var app : XCUIApplication?;
   private let AnalyticsCellIndex : UInt = 0;
 
+  private let kDidSentEventText : String = "Sent event occurred";
+  private let kDidFailedToSendEventText : String = "Failed to send event occurred";
+  private let kDidSendingEventText : String = "Sending event occurred";
```

Comment:
I use [this file](https://github.com/Microsoft/mobile-center-sdk-dotnet/blob/develop/Tests/Contoso.Forms.Test.Strings/Strings.cs#L118) as example